index.bs

```diff
@@ -606,11 +606,9 @@ dictionary SensorOptions {
 };
 
 enum SensorState {
```

Comment:
I think we want to get rid of the enum altogether, but that can be fixed in a subsequent PR.